docs/cloud/managing-airbyte-cloud.md

```diff
@@ -70,3 +70,41 @@ Understanding the following limitations will help you better manage Airbyte Clou
 * Size of a single record: 100MB
 * Shortest sync schedule: Every 60 min
 * Schedule accuracy: +/- 30 min
+
+## Buy Credits
+
+This section guides you through purchasing credits on Airbyte Cloud. An Airbyte [credit](https://docs.airbyte.com/cloud/core-concepts/#credits) is a unit of measure used to pay for Airbyte resources when you run a sync. See [Pricing](https://airbyte.com/pricing) for more information.
+
+ To buy credits:
+
+1. On the [Airbyte Cloud](http://cloud.airbyte.io) dashboard, click the **star icon**.
+    
+    The Credits page displays.
+
+2. If you are unsure of how many credits you need, click **Talk to sales** to find the right amount for your team.
+
+3. Click **Buy credits**. The Stripe payment page displays.
+
+4. If you want to change the amount of credits, click Qty **200**. **Update quantity** displays, and you can either type the amount or use minus (**-**) or plus (**+**) to change the quantity. Click **Update**. 
+
+    :::note 
+    Purchase limits:
+    * Minimum: 100 credits
+    * Maximum: 999 credits
+    :::
+
+    If you would like to buy a large amount of credits or a subscription plan, reach out to [Sales](https://airbyte.com/talk-to-sales).
```

Comment:
Changed wording in this sentence since last review based on Natalie's suggestions.

Reply:
Can we say ".. buy more than 1000 credits or a subscription plan"?  
"Large amount" sounds vague.
cc @nataliekwong